lib/Test/Nginx/Util.pm

```diff
@@ -1604,6 +1604,7 @@ sub run_test ($) {
 
                             if ($UseValgrind) {
                                 warn "$name\n";
+                                sleep 0.2;
```

Comment:
This hard-coded sleeping time can be wasteful on fast machines. Alas.

Reply:
@thibaultcha We should make this configurable at least? If not on-demand? Maybe reuse the existing `TEST_NGINX_SLEEP` env?